ui/lib/core/addon/components/form-field.hbs

```diff
@@ -16,22 +16,42 @@
   {{/unless}}
   {{#if @attr.options.possibleValues}}
     {{#if (eq @attr.options.editType "radio")}}
-      <div class="control columns" data-test-radio-input>
+      <div class="control {{unless this.hasRadioSubTextAndLabel 'columns'}}" data-test-radio-input>
         {{#each (path-or-array @attr.options.possibleValues @model) as |val|}}
-          <div class="column is-narrow is-flex-center has-text-grey has-right-margin-s" data-test-input={{@attr.name}}>
+          <div
+            class="is-flex-center
+              {{if
+                this.hasRadioSubTextAndLabel
+                'has-top-padding-xs has-bottom-padding-s'
+                'column is-narrow has-text-grey has-right-margin-s'
+              }}"
+            data-test-input={{@attr.name}}
+          >
             <RadioButton
               class="radio"
               name={{@attr.name}}
               id={{or val.value val}}
               value={{or val.value val}}
               @value={{or val.value val}}
               @onChange={{this.setAndBroadcast}}
-              @groupValue={{(get @model this.valuePath)}}
+              @groupValue={{get @model this.valuePath}}
               @disabled={{and @attr.options.editDisabled (not @model.isNew)}}
+              data-test-radio={{or val.value val}}
             />
-            <label for={{or val.value val}} class="has-left-margin-xs">
-              {{or val.value val}}
-            </label>
+            <div class="has-left-margin-xs">
+              <label
+                for="{{or val.value val}}"
+                class="has-left-margin-xs {{if this.hasRadioSubTextAndLabel 'has-text-black is-size-7'}}"
```

Comment:
I don't think the label classes should change depending on whether or not subtext exists. I think this is a good opportunity to match HDS styles and have the label just be black. Also, if a form has some radio buttons with subtext and some without, we'll want all the radio labels to be styled the same. 
```suggestion
                class="has-left-margin-xs"
```

Reply:
To confirm - it looks like other uses of `RadioButton` have black text so this class update would help match the rest of the codebase, too!

Reply:
Makes sense! Wasn't sure if we wanted to keep our current styles.

Reply:
Totally!